src/Components/Posts.js

```diff
@@ -0,0 +1,70 @@
+import { useNavigate } from 'react-router-dom'
+
+import {
+	MdiCardsHeartOutline,
+	MdiCommentMultipleOutline,
+	PhShareNetwork,
+	MaterialSymbolsArchiveOutline,
+	CarbonBookmarkAdd,
+} from '../Icones'
+
+import { StyledPost } from '../Components'
+
+const Posts = props => {
+	const { post, handleLikes } = props
```

Comment:
suggestion: you can directly do `const Posts = ({ post, handleLikes }) => ...`

Reply:
Okay, thanks for pointing this but this too looks clean in case later I have to pass many props